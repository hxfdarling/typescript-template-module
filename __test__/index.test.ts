import sum from '../src/index';
describe('test suites', () => {
  it('test module', async () => {
    expect('module').toMatchSnapshot('module');
  });
  it('test sum', () => {
    expect(sum(1, 2)).toEqual(3);
  });
});
