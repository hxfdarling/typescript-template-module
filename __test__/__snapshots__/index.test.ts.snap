// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test suites test module: module 1`] = `"module"`;
